patch/0001-arm64-dts-marvell-Add-Wistron-ES1227-54TS-board.patch

```diff
@@ -0,0 +1,423 @@
+From d44e05549493db51224447a9838b055671c19bd3 Mon Sep 17 00:00:00 2001
+From: Cody_Chen <cody_chen@wistron.com>
+Date: Tue, 25 Jun 2024 13:32:56 +0800
+Subject: [PATCH 1/2] Add Wistron dts
+
```

Comment:
Please add a commit message body, so that the patch could be upstreamed, and a Signed-off-by line.